### PR TITLE
Create a new template for Assembly.

### PR DESCRIPTION
**Pull request by jodyscho**

### CODE REVIEW

File: vip-templates/RxVIP.xctemplate/___FILEBASENAME___Assembly.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Swinject
```
Comment (jodyscho) [inline vip-templates/RxVIP.xctemplate/___FILEBASENAME___Assembly.swift]: Yes this is a duplicate file. Unfortunately, in order to create the file on it's own and as part of a full scene, we need to duplicate the files.

**Comment by matcartmill** [comment]
What's the purpose for this? What does it accomplish? Approving, but would love a bit of context. :)

**Comment by jodyscho** [comment]
@matcartmill, It's for swinject and will eventually replace the configurators. This will inject the router, presenter and interactor using swinject instead.

File: vip-templates/Assembly.xctemplate/___FILEBASENAME___Assembly.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Swinject
+import SwinjectStoryboard
+
+class ___VARIABLE_sceneName___Assembly: Assembly {
+
+    func assemble(container: Container) {
+        container.storyboardInitCompleted(___VARIABLE_sceneName___ViewController.self) { r, controller in
```
Comment (bsarrazin) [inline vip-templates/Assembly.xctemplate/___FILEBASENAME___Assembly.swift]: When did we decide to move to Swinject Storyboards?

**Pull request closed without merging**